pkg/cephfs/controllerserver.go

```diff
@@ -235,10 +239,19 @@ func (cs *ControllerServer) DeleteVolume(ctx context.Context, req *csi.DeleteVol
 	}
 	defer cr.DeleteCredentials()
 
+	// lock out parallel delete operations
```

Comment:
Locking on `VolID` is not needed, if you are locking on `RequestName` just below. If you want to fail fast, before determining the 'RequestName`, then this lock is useful, and needs to move to line 214